### DISCUSSION TITLE
You can use a special program to create a deb package.

### DISCUSSION THREAD

**Discussion by Dumitru88888** [linuxdeepin/developer-center#10753] [Developer Discussion | 开发者讨论]
You can use a special program to create a deb package. https://debreate.github.io/

**Comment by BLumia** [comment]
Some other approach to create deb package that you might interested:

- [makedeb](https://www.makedeb.org/)
- [CPack DEB Generator](https://cmake.org/cmake/help/latest/cpack_gen/deb.html)
